dist/ramda.js

```diff
@@ -4440,6 +4440,43 @@
         }, [], fns);
     };
 
+    // The algorithm used to handle cyclic structures is heavily
+    // inspired by underscore.js' implementation of isEqual
+    var _eqDeep = function _eqDeep(a, b, stackA, stackB) {
+        var idx = stackA.length;
+        while (--idx >= 0) {
```

Comment:
This might be a good time to talk about that style.  I agree that there's no good reason for `while (idx--)`.

Of course, I used to love for loops like this so, maybe my aesthetic taste is not to be trusted:

    for (var idx = arr.length; idx --> 0;) { /* ... */ }

"But it *looks* like an arrow, isn't that cool?"  :blush: